### PR TITLE
Add SCIM integration docs

### PR DESCRIPTION
**Pull request by juacker**
Adding SCIM docs.

### CODE REVIEW

File: integrations/cloud-authentication/metadata.yaml [hunk 1]
```diff
@@ -103,3 +103,71 @@
...
+      ### User Keying Between SCIM and OIDC
+      Our SCIM (System for Cross-domain Identity Management) integration utilizes OIDC (OpenID Connect) to authenticate users.
+      To ensure users are correctly identified and authenticated between SCIM and OIDC, we use the following mapping:
+
+      - SCIM externalID ↔ OIDC sub
+
+      This mapping ensures that the identity of users remains consistent and secure across both systems.
+
+      **Important**: Ensure that your OIDC and SCIM systems follow this mapping strictly.
+      The externalID in SCIM must correspond to the sub field in OIDC. Any deviation from this mapping may result
```
Comment (ilyam8) [inline integrations/cloud-authentication/metadata.yaml:169]:
```suggestion
      The externalID in SCIM must correspond to the subfield in OIDC. Any deviation from this mapping may result
```

File: integrations/cloud-authentication/metadata.yaml [hunk 2]
```diff
@@ -103,3 +103,71 @@
...
+      - The Space must be on a paid plan
+      - OIDC/SSO integration must already be enabled in one of your Spaces
+
+      ### Netdata Configuration Steps
+      1. Click on the Space settings cog (located above your profile icon).
+      2. Click on the **User Management** section and access **Authentication and Authorization** tab.
+      3. In the SCIM card, click on **Activate**.
+      4. Depending on your situation:
+         - If OIDC/SSO integration is already enabled in your Space, click **Activate**.
+         - If you have an existing SCIM integration in another Space and want to create a new one in this space linked to it, provide the SCIM token you received when enabling the first SCIM integration, and click **Activate**.
```
Comment (ilyam8) [inline integrations/cloud-authentication/metadata.yaml:134]:
```suggestion
         - If you already have a SCIM integration in another Space and want to create a linked integration here, enter the SCIM token from the original integration and click **Activate**.
```

**Comment by juacker** [comment]
@ilyam8 front end has been released, this PR is ready for review.

File: integrations/cloud-authentication/metadata.yaml (hunk 1 above)
Comment (ilyam8) [inline integrations/cloud-authentication/metadata.yaml:169]:
```suggestion
      The externalID in SCIM must correspond to the subfield in OIDC. Any deviation from this mapping may result
```

File: integrations/cloud-authentication/metadata.yaml (hunk 2 above)
Comment (ilyam8) [inline integrations/cloud-authentication/metadata.yaml:134]:
```suggestion
         - If you already have a SCIM integration in another Space and want to create a linked integration here, enter the SCIM token from the original integration and click **Activate**.
```

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**